sklearn/datasets/base.py

```diff
@@ -144,11 +144,10 @@ def load_files(container_path, description=None, categories=None,
         contains characters not of the given `encoding`. Passed as keyword
         argument 'errors' to bytes.decode.
 
-    random_state : int, RandomState instance or None, optional (default=0)
-        If int, random_state is the seed used by the random number generator;
-        If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used
-        by `np.random`.
+    random_state : int, RandomState instance or None (default)
```

Comment:
`default` -> `default = 0`

Reply:
Correct me if I'm wrong here but isn't the default `None`, which uses the global random state from `numpy.random`? Which is distinct from `random_state=0` which would set the seed to zero.

Reply:
Here it is 0

Reply:
Step 1. Remove foot from mouth.
Step 2. Implement the obvious and reasonable fixes that the devs told you to.

Thanks @qinhanmin2014 and @jnothman for pointing that out. Sorry for my confusion. The latest commits fix it.